spec/rubocop/cop/rspec/repeated_example_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::RSpec::RepeatedExample do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated example' do
+    expect_violation(<<-RUBY)
+      describe 'doing x' do
+        it "does x" do
+                    ^^ Don't repeat examples within an example group.
+          expect(foo).to be(bar)
+        end
+
+        it "does y" do
+                    ^^ Don't repeat examples within an example group.
+          expect(foo).to be(bar)
+        end
+      end
+    RUBY
+  end
+
+  it 'does not register a violation if rspec keyword magic is involved' do
+    expect_no_violations(<<-RUBY)
+      describe 'doing x' do
+        it "does x" do
+          expect(foo).to be(bar)
+        end
+
+        it "does y", :magic do
```

Comment:
Fair. I used `:magic` to make it a bit more clear that what we're avoiding is tests that magically import special behavior using RSpec's tags.

Also this is definitely not a nitpick (I want this feedback). Matching rspec's language in this codebase is pretty helpful